AmericanMedicalAssociation/joe-style-guide
# EWLJ-556: Add background header image to Multimedia page and filter page

## JIRA Ticket(s)
- [EWLJ-556: Add background header image to Multimedia page and filter page](https://issues.ama-assn.org/browse/EWLJ-556)


## Description

- Adds background image for path matching `/multimedia` to image assets folder


## To Test

- Run `gulp serve` 
- Setup local style guide on Drupal, if not done
- Clear cache
- Ensure listing pages backgrounds are rendering on Drupal's `/multimedia` page: http://ama-joe.local/multimedia
- Drupal PR: https://github.com/AmericanMedicalAssociation/ama-d8/pull/2227

## Visual Regressions

n/a

## Relevant Screenshots/GIFs

![Screen Shot 2020-08-31 at 10 34 46 AM](https://user-images.githubusercontent.com/541745/91732053-af1f8180-eb75-11ea-95b8-800398e1792d.png)



## Remaining Tasks

Remaining tasks?


## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?
